sql/core/src/main/scala/org/apache/spark/sql/execution/python/AggregateInPandasExec.scala

```diff
@@ -94,6 +94,8 @@ case class AggregateInPandasExec(
   }
 
   override protected def doExecute(): RDD[InternalRow] = {
+    metrics // force lazy init at driver
```

Comment:
do we still need it after https://github.com/apache/spark/pull/33559/files#diff-c739b6441208b9f12c613d0ba8146fef45b4553527c6aa9626d42bad0b1df847R25 ?

Reply:
Good point, I have now removed  it, thanks.